components/gcp/bigquery/query/component.yaml

```diff
@@ -16,6 +16,9 @@ name: Bigquery - Query
 description: |
   A Kubeflow Pipeline component to submit a query to Google Cloud Bigquery 
   service and dump outputs to a Google Cloud Storage blob. 
+metadata:
+  labels:
+    add-pod-env: 'true'
```

Comment:
Maybe we should have namespace prefix in the key name.

Reply:
I feel this label is pretty common and not necessary to be KFP specific. Ideally, we should have a admission control which inject it to all pods in kf cluster.